src/components/aboutPage/OurStory.jsx

```diff
@@ -7,8 +7,11 @@ export default function OurStory() {
         <h1 className='mt-[37px] md:mt-[126px] mb-[14px] md:mb-[43px] text-[32px] md:text-[64px] text-[#025594] font-bold leading-[38px] md:leading-[75px]'>Our Story</h1>
         </div>
         <div className='mb-[50px] md:mb-[129px]'>
-         <p className='max-w-[312px] md:max-w-[817px] text-center text-[16px] md:text-[24px] font-normal leading-[18.75px] md:leading-[28px]'>
-          It is a long established fact that a reader will be distracted by the readable content of a page when looking at its layout. The point of using Lorem Ipsum is that it has a more-or-less normal distribution of letters, as opposed to using Content here, content here, making it look like readable English.
+         <p className='max-w-[312px] md:max-w-[817px] text-center text-[16px] md:text-[24px] font-normal leading-4 md:leading-[36px]'>
```

Comment:
in mobile view, text still looks shrinked and line heights between texts too close,.